### PR TITLE
Endpoint

### PR DESCRIPTION
**Pull request by iambenkis**
### Description :desktop_computer:  [(Frontend PR) ](https://github.com/iambenkis/Hello-world-frontend/pull/1)
In this milestone I created an API that respect the following requirements.

### Project requirements :1st_place_medal: 
 - Create a new Rails API app called 'hello-rails-back-end'. :heavy_check_mark: 
 - Initialize your project with Git. :heavy_check_mark: 
 - Make sure that your project has a Postgres database set up.  :heavy_check_mark: 
 - Create a table for storing your messages and create 5 different greetings.  :heavy_check_mark: 
 - Create an API endpoint that selects a random greeting from your table (you will need a controller with an action an rails route). :heavy_check_mark:

### CODE REVIEW

**Review by arslanbisharat — COMMENTED** [review]
Hi @iambenkis, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

The README file for the fronted should have a link to the backend part. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by iambenkis** [comment]
> Hi @iambenkis,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> The README file for the fronted should have a link to the backend part.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

@arslanbisharat ,You should Always check twice :eyes:  before invalidating :smile:

File: README.md [hunk 1]
```diff
@@ -1,24 +1,274 @@
...
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (addod19) [inline README.md:272]: - [ ] It would be better if we fix the broken link to our MIT license file, it takes us to 404 page

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @iambenkis, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] README file of both frontend and backend has its corresponding link to the PR :100: 
- [ ] Correct Github-flow followed :fireworks: 
- [ ] Frontend communicates successfully with the backend api :rocket: except that our database is empty
- [ ] Controller and routes matches with the url :1st_place_medal: 
- [ ] Migration file and model present :fireworks: 
- [ ] README file contains all the required headers :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Seeding our database so that we can easily get our random message when we interact with the frontend, otherwise we have to manually create data in our backend before interacting with the frontend successfully :man_technologist: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (addod19) [inline README.md:272]: - [ ] It would be better if we fix our MIT license link, the link is broken :man_technologist:

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi  @iambenkis, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] README file of both frontend and backend has its corresponding link to the PR :100: 
- [ ] Correct Git-flow followed :fireworks: 
- [ ] Frontend communicates successfully with the backend api :rocket: 
- [ ] App and Greeting component present :1st_place_medal: 
- [ ] README file contains all the required headers :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Benjamin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**